app/local/page.tsx

```diff
@@ -0,0 +1,18 @@
+import FeedPage from "@/components/feed/localindex";
+import PageTitle from "@/components/ui/PageTitle";
+import { getFeedItems } from "@/lib/feed";
+import { getCurrentUser } from "@/lib/getUser";
+
+const Feed = async () => {
+  const data = await getFeedItems();
```

Comment:
All of the filtering should be done here (ideally inside of a lib function) and then passed into the same `FeedPage` component (aka deleting the `localIndex` file)